glean-core/csharp/GleanTests/DispatchersTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using Xunit;
+
+namespace Mozilla.Glean.Tests
+{
+    public class DispatchersTests
+    {
+        [Fact]
+        public async void TestTaskQueueing()
```

Comment:
Are you planning on porting over all the tests from Kotlin?

Reply:
Yes, I only had one test complete when I asked you to take a look, but the plan is to have the exact same coverage and tests.